Sources/ComposableArchitecture/Store.swift

```diff
@@ -768,7 +823,7 @@ public typealias StoreOf<R: ReducerProtocol> = Store<R.State, R.Action>
     func rescope<ScopedState, ScopedAction, RescopedState, RescopedAction>(
       _ store: Store<ScopedState, ScopedAction>,
       state toRescopedState: @escaping (ScopedState) -> RescopedState,
-      action fromRescopedAction: @escaping (RescopedAction) -> ScopedAction,
+      action fromRescopedAction: @escaping (RescopedState, RescopedAction) -> ScopedAction?
```

Comment:
This appears to be missing a comma?
```suggestion
      action fromRescopedAction: @escaping (RescopedState, RescopedAction) -> ScopedAction?,
```